tt_metal/tools/profiler/kernel_profiler.hpp

```diff
@@ -167,23 +164,21 @@ namespace kernel_profiler{
             profiler_control_buffer[FLAT_ID] = core_flat_id;
         }
 
-#pragma GCC unroll 65534
         for (int i = GUARANTEED_MARKER_1_H; i < CUSTOM_MARKERS; i ++)
         {
-        //TODO(MO): Clean up magic numbers
+            //TODO(MO): Clean up magic numbers
```

Comment:
where are these numbers getting used? Is there risk that device already holds these values on start, and FW jumps ahead of init?

Reply:
I am referring to the `0x80000000`. They are pretty settled at this point. I will clean them up in another PR

Reply:
We are in init, so we are flushing out anything that might be there from the previous run.